src/Raven.Server/Documents/Indexes/CollectionOfBloomFilters.cs

```diff
@@ -441,6 +477,14 @@ public void Flush()
                 _tree.Increment(_keySlice, Count - _initialCount);
             }
 
+            internal void Delete()
+            {
+                if (_tree.Llt.Environment.Options.IsCatastrophicFailureSet)
```

Comment:
Under what scenario this will happen?

Reply:
I copied it from Flush() method,
The scenario of Delete() is similar to Flush() both called from same place.
https://github.com/ravendb/ravendb/pull/10329/files#diff-bbed84f00f70fe07cd5deb3915a02d4aR474